### PR TITLE
Helen/refactor

### PR DESCRIPTION
**Pull request by HacksForHugs**
There is a lot of stuff in here but mostly:

- [ ] Building operators log in and initial state is working fine, there is a separate issue we have not yet addressed.
- [ ] Asset manager log in and initial state is working fine
- [ ] Rmi user log in and initial state is working fine
- [ ] Export CSV actually populates answers
- [ ] Rake db:seed works. RUN RAKE DB RESET THE FIRST TIME pls don't seed on top of any existing seeds
- [ ] Added pry to debug, it works in rails console and in code.
- [ ] Added figaro to manage environment variables because bad unix. Everyone is gonna need my application.yml

### CODE REVIEW

File: app/models/dropdown_option.rb [hunk 1]
```diff
@@ -10,6 +10,6 @@
 #
 
 class DropdownOption < ApplicationRecord
-  has_one :child_question, class_name: 'Question', as: :parent_option, :dependent => :destroy
+  has_many :child_questions, class_name: 'Question', as: :parent_option, :dependent => :destroy
```
Comment (tonyyanga) [inline app/models/dropdown_option.rb]: @HacksForHugs Should RangeOption be kept consistent, i.e. has_many child questions? I would also need to change my PR code.

File: app/javascript/packs/actions/answers.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+export async function addAnswer(buildingId, answer, dispatch) {
+  dispatch(createAnswer(buildingId));
+
+  try {
+    const answerData = JSON.stringify({
+      ...answer
+    });
+
+    let response = await fetch('/api/buildings', {
```
Comment (tonyyanga) [inline app/javascript/packs/actions/answers.js]: Should answer be created via `/api/buildings` or `/api/answers`?

File: app/models/dropdown_option.rb (hunk 1 above)
Comment (HacksForHugs) [reply app/models/dropdown_option.rb]: waitaminute i do not recall ever making this change

File: app/javascript/packs/actions/answers.js (hunk 2 above)
Comment (HacksForHugs) [reply app/javascript/packs/actions/answers.js]: oh nooooOOOO i think i rebased or something this isn't my commit

Comment (tonyyanga) [reply app/javascript/packs/actions/answers.js]: RIP

**Pull request merged**